internal/db/pg/pets.go

```diff
@@ -82,3 +82,19 @@ func (r *Repository) UpdatePet(ctx context.Context, p *gostreamv1.Pet) error {
 	}
 	return nil
 }
+
+// DeletePet ...
+func (r *Repository) DeletePet(ctx context.Context, p uint64) error {
+	query, params, err := sq.Delete("pets").Where("id = ?", p).
+		PlaceholderFormat(sq.Dollar).ToSql()
+	if err != nil {
+		return errs.B().Code(errs.Internal).Msg("sq error").Err()
+	}
+	if _, err = r.pool.Exec(ctx, query, params...); err != nil {
+		if errors.Is(err, pgx.ErrNoRows) {
+			return errs.B().Code(errs.InvalidArgument).Msg("pet does not exist").Err()
+		}
+		return err
+	}
```

Comment:
like in DeleteUser, wrap this with internal error code.

```
return errs.WrapCode(err, errs.Internal, "database error occurred")
```

Reply:
Everything else looks good

Reply:
fixed